packages/design-tokens/README.md

```diff
@@ -153,10 +153,12 @@ module: {
         {
           loader: 'sass-loader',
           options: {
+            api: 'modern-compiler',
```

Comment:
I think it would be fine to add a link to the SASS documentation here about the usage of the new `sass-embedded` package. Also, this required also `sass-embedded` to be in a `package.json` IMHO.

Reply:
Or even mention that there are legacy and modern APIs and also a modern compiler supporting those.

Reply:
made some changes to the readme, do you think its better to have both options like this?

Reply:
Nice, much better :-)